meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/supervisor.conf

```diff
@@ -1,3 +1,4 @@
+# This file represents the last known version of the supervisor
```

Comment:
@agherzan I don't want to change how this works. I only put this comment there so that any user of this data knows what to expect from it.